osc-ui/src/main/java/org/osc/core/broker/view/CRUDBaseView.java

```diff
@@ -68,7 +68,7 @@
  */
 public abstract class CRUDBaseView<P extends BaseDto, C extends BaseDto> extends VerticalLayout implements View {
 
-    //private static final Logger log = Logger.getLogger(CRUDBaseView.class);
+    //private static final Logger log = LogComponent.getLogger(CRUDBaseView.class);
```

Comment:
should this be log provider or log component?

Reply:
Should be provider `LogProvider.getLogger(DistributedApplianceView.class)` but just delete the commented out line.

Reply:
I'd say it should be deleted